packages/web-components/src/components/va-alert/va-alert.tsx

```diff
@@ -156,7 +156,11 @@ export class VaAlert {
       <Host>
         <div role={role} aria-live={ariaLive} class={classes}>
           <i aria-hidden="true" role="img"></i>
-          <div class="body" onClick={this.handleAlertBodyClick.bind(this)}>
+          <div
+            class="body"
+            onClick={this.handleAlertBodyClick.bind(this)}
+            role="presentation"
```

Comment:
This fixes the issue from the component-library side of things. Any React applications that consume this component will need to be updated with role="presentation" to prevent NVDA from announcing the element as clickable.

Reply:
What do you mean by consuming React apps will need to be updated? If an app has this:

```jsx
return (
  <div>
    <va-alert>
      Alert content
    </va-alert>
  <div>
  );
```

Will it need to be updated to this?

```jsx
return (
  <div>
    <va-alert role="presentation">
      Alert content
    </va-alert>
  <div>
  );
```

Or do they just need to be updated to the published version with this fix?

Reply:
I was incorrect in my previous comment. **This issue is now fixed completely at the component level.** NVDA won't read this element as "clickable, heading level 3, ..." with `role="presentation` on the div with the onClick.

There is another bug related to React and NVDA where NVDA reads the React parent container as "clickable" even if there is no interactive content. This is easily testable in Storybook or [this example codesandbox](https://codesandbox.io/s/epic-babbage-qc9xj). The fix for that is to add `role="presentation"` to the root element where React mounts.

---

NVDA reads this as "clickable, heading level 2".
```jsx
const rootElement = document.getElementById("root");
ReactDOM.render(
  <StrictMode>
    <App />
  </StrictMode>,
  rootElement
);

<div className="App">
      <h2>This is a heading within a React application.</h2>
</div>
```

If `#root` has `role="presentation"` set, NVDA correctly reads this as "heading level 2".

---

I was able to fix this issue in Storybook by adding this code snippet below in `.storybook/preview.js`. I've removed it from this PR because this change only affects stories, but I can add it back in until it's fixed by React/NVDA. 

```jsx
document.body.onload = function () {
  document.querySelector('#root').setAttribute('role', 'presentation');
};
```

The work for https://github.com/department-of-veterans-affairs/va.gov-team/issues/34161 is completed with the changes in this PR. The remaining React/NVDA issue should not be part of this PR. We should create a new issue for that.

Reply:
Should I re-add the Storybook fix for this issue? It would make it easier to test these types of issues while we wait for the official React/NVDA fix.

Reply:
I think making sure storybook reads correctly is a good idea. We could have that `onload` handler in the `preview.js`, but storybook also gives us a way to define our own custom root and put whatever tags we want on it:

https://storybook.js.org/docs/react/configure/story-rendering#adding-to-body